.circleci/config.yml

```
@@ -26,7 +26,7 @@ workflows:
       - nightly-wagtail-test
     triggers:
       - schedule:
-          cron: "0 0 * * *"
+          cron: "0 0 * 1 *"
```

According to https://crontab.guru/#0_0_*_1_*, this cron runs at midnight every 1st of January. 

Perhaps you meant to do this? This is cron runs at midnight every Monday. See https://crontab.guru/#0_0_*_*_1
```suggestion
          cron: "0 0 * * 1"
```

---

d'oh! thank you Storm